deploy-board/deploy_board/webapp/helpers/environ_hosts_helper.py

```diff
@@ -30,3 +30,36 @@ def get_host_by_env_and_hostname(request, env_name, stage_name, host_name):
 def stop_service_on_host(request, env_name, stage_name, host_ids):
     return deploy_client.delete("/envs/%s/%s/hosts" % (env_name, stage_name), request.teletraan_user_id.token,
                                 data=host_ids)
+
+
+def get_host_tags(request, env_name, stage_name, tag_name):
+    return deploy_client.get("/envs/%s/%s/host_tags/%s" % (env_name, stage_name, tag_name),
+                             request.teletraan_user_id.token)
+
+
+def remove_host_tags(request, env_name, stage_name, tag_name):
+    return deploy_client.delete("/envs/%s/%s/host_tags/%s" % (env_name, stage_name, tag_name),
+                                request.teletraan_user_id.token)
+
+
+def get_ec2_host_tags(request, env_name, stage_name, tag_name):
```

Comment:
It sounds a little bit weird this is called get_ec2_host_tags with a reload=true parameter